src/pages/evm/wallet/WalletTransactionRow.vue

```diff
@@ -129,7 +129,7 @@ export default defineComponent({
             return `${this.chainSettings.getExplorerUrl()}/tx/${this.transaction.id}`;
         },
         longDate(): string {
-            return getLongDate(this.transaction.epoch);
+            return getFormattedDate(this.transaction.epoch, 'MMM d, yyyy hh:mm:ss a', true);
```

Comment:
in src/pages/evm/staking/WithdrawRaw.vue:44 you are using a constant for the date string.
Maybe you can use another constant here to be consistent.